test/support/builders/atom_watcher_event.js

```diff
@@ -0,0 +1,70 @@
+const metadata = require('../../../core/metadata')
+const events = require('../../../core/local/steps/event')
+
+/*::
+import type { Stats } from 'fs'
+import type { AtomWatcherEvent, Action, Kind, Batch } from '../../../core/local/steps/event'
+*/
+
+function randomPick (elements /*: Array<T> */) /*: T */{
+  const l = elements.length
+  const i = Math.floor(Math.random() * Math.floor(l))
+  return elements[i]
+}
+
+module.exports = class AtomWatcherEventBuilder {
+  constructor () {
+    this._event /*: AtomWatcherEvent */ = {
+      action: randomPick(events.ACTIONS),
+      kind: randomPick(events.KINDS),
+      path: '/',
+      _id: '/'
+    }
+  }
```

Comment:
What is the purpose for a stochastic constructor?

Reply:
g

I want to make sure we don't rely on the action and kind of an event unless we've specified it.
It's also a way to test more combinations over time.